packages/framework/request-handler/src/runtimeRequestHandlerBuilder.ts

```diff
@@ -13,7 +13,7 @@ import { RuntimeRequestHandler } from "./requestHandlers";
  * The RuntimeRequestHandlerBuilder creates a runtime request handler based on request handlers.
  * The provided handlers sequentially applied until one is able to satisfy the request.
  *
- * @deprecated Will be removed in future major release. Migrate all usage of IFluidRouter to the "entryPoint" pattern. Refer to Removing-IFluidRouter.md
+ * @deprecated Will be removed once Loader LTS version is "2.0.0-internal.7.0.0". Migrate all usage of IFluidRouter to the "entryPoint" pattern. Refer to Removing-IFluidRouter.md
```

Comment:
Quick search shows me this is only used by `buildRuntimeRequestHandler` below, can we stop exporting it now, even if the code isn't yet deleted?